src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/ServiceCollectionEventSubscriptionExtensionsTests.cs

```diff
@@ -233,7 +233,7 @@ private static void GivenConfigurations(IServiceCollection services, params stri
         }
 
         var configuration = new ConfigurationBuilder()
-            .AddInMemoryCollection(dictionnary)
+            .AddInMemoryCollection(dictionnary!)
```

Comment:
I think you should change the dictionary type instead (`Dictionary<string, string?>`)

Reply:
also corrected typos in variable name